source/entities/EntityBase.h

```diff
@@ -127,6 +128,15 @@ class EntityBase
     { return false; }
     virtual void drop(const Ogre::Vector3& v)
     {}
+    inline void show()
+    {
+        RenderManager::getSingleton().rrAttachEntity(this);
```

Comment:
So we sacriface game speed for the sake of compilation speed ? Ok I can un - inline that function.